campaigns/campModel.js

```diff
@@ -0,0 +1,56 @@
+const db = require('../database/dbConfig');
+
+module.exports = {
+  find,
+  findById,
+  findCampById,
+  insert,
+  update,
+  remove
+};
+
+function find() {
+  return db('campaigns');
+    .join('users', 'users.id', 'campaigns.user_id')
```

Comment:
Is using join more proficient than to nest model functions in the route? For example, find() on the user table, then nest findByUserId() on the campaign table. To then structure the object how you see fit.

Reply:
I'm not sure! My thinking is fitting the entire thing into one query means less calls to the server which = faster. Plus, joins are cool?

Reply:
Definitely I can see that calling the server one time can be a plus, I also can see having a large object one set of columns on it somewhat confusing. I think it's a matter of preference to be honest.